lib/active_merchant/billing/gateways/fusebox.rb

```diff
@@ -129,7 +130,12 @@ def inquiry(fields)
       end
 
       # Storing a card is done via an authorize for $0.00 with the string "ID:" in the token request field
+      # NB: Some cards/banks return an error for $0 auth, so you have to use $1 auth and then reverse it.
       def store(creditcard, options = {})
+        authorize(0, creditcard, options.merge(:token_request => 'ID:'))
+      end
+
+      def store_and_auth(creditcard, options = {})
```

Comment:
You were right.  I made them different methods because the docs suggested they did different things under the hood.

But they're the same method in the Fusebox API, so I changed this to your suggestion of `auth_amount: 100` as an option.